packer.mk

```diff
@@ -43,8 +40,13 @@ else ifeq (${OS_SKU},CBLMariner)
 	@echo "Using packer template file vhd-image-builder-mariner.json"
 	@packer build -var-file=vhdbuilder/packer/settings.json vhdbuilder/packer/vhd-image-builder-mariner.json
 else ifeq (${OS_SKU},AzureLinux)
+ifeq ($(findstring cvm,$(FEATURE_FLAGS)),cvm)
```

Comment:
After the PR https://github.com/Azure/AgentBaker/pull/5797 was merged, this PR should be rebased to get the latest master.